README.md

```diff
@@ -87,7 +87,7 @@ Awesome libraries, programs, and mods concerning ComputerCraft and its forks.
 
 ### Charts and tables
 
-- [Cake's hex font chart](https://thox.madefor.cc/_images/encodings-cc-chars.png) and [Wojbie's decimal font chart](https://cdn.discordapp.com/attachments/477911902152949771/933498000385400862/1642633650325141456271.png): Contains decimal and hex mappings to ComputerCraft font characters.
```

Comment:
I don't think we need to separate these.

Reply:
These are not separated, just flipped around to make sense with the following text.

Reply:
@SkyTheCodeMaster why were these separated?

Reply:
Oh, alright. The diff made it look odd.